keyboards/helix/rules.mk

```
@@ -32,6 +32,6 @@ RGBLIGHT_ENABLE = no    # Enable WS2812 RGB underlight.
 # Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
 SLEEP_LED_ENABLE = no   # Breathing sleep LED during USB suspend
 
-DEFAULT_FOLDER = helix/rev2
+DEFAULT_FOLDER = helix/rev3_5rows
```

Changing this will require it to go via develop.

```suggestion
DEFAULT_FOLDER = helix/rev2
```

---

Please tell me a little more detail.
When do you think I should change it?

---

@MakotoKurauchi, I think what zvecr is talking about is the following.

* If you just add `helix/rev3_4rows/` and `helix/rev3_4rows/`, you can merge them into the `master` branch.
* Changes to `helix/rules.mk` are treated as "breaking change" because they affect existing `helix/rev2/` and `helix/pico/`, so they need to be merged into the `develop` branch.

(わたしは、'via' を 'VIA' と捉えて、最初、意味がわかりませんでした。)

---

I understand that changing the default is not easy.
We plan to make our keyboard under the `yushakobo` directory in the future.
Do you think this rev3 should be `yushakobo/helix/rev3_5rows/` and `yushakobo/helix/rev3_4rows/`?

---

Personally, I think the following directory structure is fine for the short term.

* `keyboards/`
  * `helix/` (For the time being, keeping the old configuration.)
  * `yushakobo/`
    * `helix/` (Added with this PR)
      * `rev3_4rows/`
        * `keymaps/`
      * `rev3_5rows/`
        * `keymaps/`

---

However, personally, in the long run, I think the following directory structure is better.
( This is just a suggestion. I think the decision is in MakotoKurauchi's hands.)

* `keyboards/`
  * `helix/` (For the time being, keeping the old configuration.)
  * `yushakobo/`
    * `helix_4rows/` (Added with this PR)
      * `keymaps/`   (Helix series 4-rows type common keymaps)
      * `rev3/`      (Added with this PR)
      * `rev2/`      (Add this later in another PR. `split_common` base)
      * `pico/`      (Add this later in another PR. `split_common` base)
    * `helix_5rows/` (Added with this PR)
      * `keymaps/`   (Helix series 5-rows type common keymaps)
      * `rev3/`      (Added with this PR)
      * `rev2/`      (Add this later in another PR. `split_common` base)